src/cli/flags/restore_config.go

```diff
@@ -25,5 +27,8 @@ func AddRestoreConfigFlags(cmd *cobra.Command) {
 		"Sets the behavior for existing item collisions: "+string(control.Skip)+", "+string(control.Copy)+", or "+string(control.Replace))
 	fs.StringVar(
 		&DestinationFV, DestinationFN, "",
-		"Overrides the destination where items get restored; '/' places items into their original location")
+		"Overrides the folder where items get restored; '/' places items into their original location")
+	fs.StringVar(
+		&ToResourceFV, ToResourceFN, "",
+		"Overrides the protected resource (mailbox, site, etc) where data gets restored")
```

Comment:
```suggestion
		"Overrides the protected resource (mailbox, site, user, etc) where data gets restored")
```

Reply:
Isn't that covered in `etc`? 😆

Reply:
IMO, it's important enough to not be in `etc`